Makefile

```
@@ -9,9 +9,11 @@ clean:
 
 .PHONY: coverage
 coverage:
-	pytest -qq
+	python3 -c "import pytest" || python3 -m pip install pytest
```

Let's redirect all these to `/dev/null` to avoid the error message:

```
python3 -c "import pytest" || python3 -m pip install pytest
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'pytest'
Collecting pytest
...
```


```suggestion
	python3 -c "import pytest" > /dev/null 2>&1 || python3 -m pip install pytest
```

---

Sure, done